packages/components/psammead-sitewide-links/src/index.jsx

```diff
@@ -61,11 +71,19 @@ const linkPropTypes = shape({
   text: string.isRequired,
 });
 
+const trustProjectLinkPropTypes = shape({
+  href: string,
+  text: string,
+});
+
 SitewideLinks.propTypes = {
   links: arrayOf(linkPropTypes.isRequired).isRequired,
   copyrightText: node.isRequired,
+  trustProjectLink: oneOfType([trustProjectLinkPropTypes, bool]),
```

Comment:
trustProjectLink shouldn't need to have type boolean.
```suggestion
  trustProjectLink: linkPropTypes,
```

Reply:
resolved